.github/workflows/multimodal-pr.yml

```
@@ -107,3 +100,41 @@ jobs:
           ipython kernel install --name "python3" --user
           echo "Current branch: ${BRANCH_NAME}"
           papermill --parameters repo_branch ${BRANCH_NAME} notebooks/phrase_grounding.ipynb /tmp/phrase_grounding_output.ipynb
+
+  publish:
+    runs-on: ubuntu-18.04
+    needs: [ flake8, mypy, pytest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Install pypa/build
```

pypa?

---

That was copy & pasted. I guess it refers to https://github.com/pypa/build.

---

I don't think the pypa prefix adds value.
```suggestion
      - name: Install build tools
```